### DISCUSSION TITLE
Update Monaco Editor

### DISCUSSION THREAD

**Discussion by JonasHelming** [eclipse-theia/theia#10732] [Improvements] [answered]
- Update to the latest version
- Simplify the update
- Update more frequently

**Comment by colin-grant-work** [comment]
Other places for comment on this topic:
https://github.com/eclipse-theia/theia/issues/10387
https://github.com/eclipse-theia/theia/pull/10736

**Comment by JonasHelming** [comment]
@colin-grant-work : I would set this to "answered", as you did significant improvements here, fine for you?

**Comment by colin-grant-work** [comment, marked as answer]
Sounds good to me. I did close the issue closest to this (#7457) as complete with the recent uplift and the documentation added there.
